assets/css/_variables.scss

```diff
@@ -15,3 +17,10 @@ $breakpoints: (
     'xxl': 1400px
 );
 
+$padding-breakpoints: (
+  576px: 20px,
+  768px: 40px,
+  992px: 60px,
+  1200px: 100px,
+);
```

Comment:
ditto: can we use variables for breakpoints?

Reply:
resolved. Bhai. Thanks